src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java

```diff
@@ -241,11 +244,11 @@
    * To create a case event being given a parent case and json to describe the
    * case event to be created
    *
-   * @param caseId the parent case
+   * @param caseId                      the parent case
```

Comment:
Did you mean to add these tabs? They've been added to a lot of the docs not sure if it makes it easier to read or messy :/

Reply:
Think IntelliJ auto adds them